server/services/deployment/packages-sil-org.ts

```diff
@@ -12,23 +12,22 @@ or:
 
 ```
 Package: keyman
-Source: keyman-config
-Version: 14.0.273-1+focal1
+Version: 17.0.330-1+noble1
 ```
 
 */
 
 import httpget from "../../util/httpget";
 import DataService from "../data-service";
 
-// https://packages.sil.org/ubuntu/dists/focal/main/binary-amd64/Packages
+// https://packages.sil.org/ubuntu/dists/noble/main/binary-amd64/Packages
 const HOST='packages.sil.org';
-const PATH='/ubuntu/dists/focal/main/binary-amd64/Packages';
+const PATH='/ubuntu/dists/noble/main/binary-amd64/Packages';
 
 const service: DataService = {
    get: function() {
     return httpget(HOST, PATH).then((data) => {
-      const results = data.data.match(/^Package: keyman\s*\nSource:\s*keyman[^\n]*\nVersion: (\d+\.\d+\.\d+)-\d+/m);
+      const results = data.data.match(/^Package: keyman\s*\n(Source:\s*keyman[^\n]*\n)?Version: (\d+\.\d+\.\d+)-\d+/m);
       if(results) {
         return { version: results[1] };
       }
```

Comment:
```suggestion
      const results = data.data.match(/^Package: keyman\s*\n(Source:\s*keyman[^\n]*\n)?Version: (\d+\.\d+\.\d+)-\d+/m);
      if(results) {
        return { version: results[2] };
      }
```

Either update to `results[2]` or else use `?:` to make the Source group non-capture.

Reply:
right, thanks!